module/motion/QuinticWalk/src/WalkEngine.cpp

```diff
@@ -462,56 +455,59 @@ namespace module::motion {
 
         // Define trunk yaw target orientation position and velocity in euler angle and convertion to axis
         // vector
+
+        // TODO(MotionTeam): Investigate why the added pitch and yaw coef's cause the robot to drift left for forward
```

Comment:
Make an issue directing to this file

Reply:
Maybe just a have a general walk improvements issue and have any other ideas in there too